flycheck.el

```diff
@@ -10443,16 +10494,12 @@ expression, which selects linters for lintr."
   :risky t
   :package-version '(flycheck . "0.23"))
 
-(defun flycheck-r-has-lintr (R)
-  "Whether R has installed the `lintr' library."
-  (with-temp-buffer
-    (let ((process-environment (append '("LC_ALL=C") process-environment)))
-      (call-process R nil t nil
-                    "--slave" "--restore" "--no-save" "-e"
-                    "library('lintr')")
-      (goto-char (point-min))
-      (not (re-search-forward "there is no package called 'lintr'"
-                              nil 'no-error)))))
+(defun flycheck-r-has-lintr (checker)
+  "Whether CHECKER (R) has installed the `lintr' library."
+  (eq 0 (flycheck-call-checker-process
```

Comment:
nitpick:
```suggestion
  (zerop (flycheck-call-checker-process
```

Reply:
:thinking: I think that will raise an error if the return value is not a number, but call-process can return strings (and our wrapper can return nil, as well)

Reply:
That being said, eql would be better than eq here. Changed.